### DISCUSSION TITLE
helpme!

### DISCUSSION THREAD

**Discussion by iHibro** [filecoin-project/lotus#11111] [General Q&A] [answered]
Hello, I am a developer from China

I lost my Lotus server (all files), but I kept the owner private key and storage provider address.

I have created a new server and synchronized lotus and lotus miner. May I know how to proceed to view my previous sector_ Numbers, and how to resend WithdrawBalance messages

Thank you very much. I have searched through all the tutorials, but there are really few Chinese tutorials available

**Comment by beck-8** [comment, marked as answer]
1. 你只拥有owner？是否设置/有worker和control地址？如果你的owner和worker地址不一样，那么需要更改worker地址，更改需要900个高度生效，在此期间你不能出块，但是你可以使用owner或者control钱包提交wdpost消息。
2. 是否拥有之前的backup，有的话基于这个恢复，没有继续下一步
3. 运行 `lotus-miner init --actor`恢复repo
4. 启动miner，配置之前环境的 持久存储（store）,重新建立索引。这个时候不需要关心miner info里什么也没有，只需要关注索引正常，wdpost正常，出块正常。
5. 在此检查相关环境、配置，结束。
6. 如果你是想终结扇区取回FIL，那么你可以使用`lotus-shed sectors termination-estimate` 预估费用，使用`lotus-shed sectors termination` 真正的终止。（前提你知道你自己在干什么）如果你的节点很大，或者还是不知道如何计算惩罚的预估，你可以把你的节点号发出来，我帮你算。

**Reply by iHibro** [reply]
lotus-shed sectors termination 这个命令不需要扇区号么，我新恢复的可以看到或操作我以前的扇区号么

**Reply by beck-8** [reply]
新恢复的本地看不到数据。终止的扇区号从链上找
lotus state active-sectors

**Reply by iHibro** [reply]
谢谢！lotus state active-sectors f**** 有几万个结果，是需要中止哪一个呢~
